docs/authoring-components.md

```diff
@@ -64,7 +64,7 @@ project.
 
 Note that this document assumes you are familiar with the library and its
 [architecture](code/architecture.md). If that is not the case, we recommend reading that first. If you
-are brand new to the project, we recommend starting with our [general developer documentation](./developer.md).
+are brand new to the project, we recommend starting with our [Getting Started Guide](./getting-started.md).
```

Comment:
Given that this is developer-specific, maybe we should point an absolute link to either https://github.com/material-components/material-components-web/tree/master/docs/code, or https://github.com/material-components/material-components-web/blob/master/docs/code/architecture.md (since these aren't crawled for material.io)?

Honestly I find it a little unfortunate that we removed https://github.com/material-components/material-components-web/blob/v0.18.0/docs/developer.md... cc @lynnjepsen if she has any thoughts on that.

Reply:
Yeah. Developer gave some good centric place to find other docs. However, the previous line already talk about the architecture and its link and the purpose here is for "are brand new to the project"

Reply:
Hm. Yeah, I guess it can be interpreted that way. I interpreted it as brand new to developing on MDC Web itself, as opposed to brand new to using it.